### DISCUSSION TITLE
BUG: Dependency on internet connection

### DISCUSSION THREAD

**Discussion by spop** [fcakyon/yolov5-pip#244] [General]
Starting from v7.0.1 the `DetectMultiBackend` needs internet access in order to work.
If no internet access, the following line raises an error: `result = attempt_download_from_hub(w, hf_token=hf_token)`, even if a local model is given (i.e. a .pt file).
The line is not present in the `https://github.com/ultralytics/yolov5` .
The v7.0.0 is the last version to work.
Could you fix it in a new release?
Thank you.

**Comment by lumino33** [comment]
I got the same problem. Have you addressed this? 
I saw the option 'pretrained' in the docstrings but not in code :((((
![Screenshot from 2025-02-04 16-45-43](https://github.com/user-attachments/assets/e12b5fc4-7e48-4b89-90f1-830ada760d46)
